src/plugins/data/server/search/session/session_service.ts

```diff
@@ -208,10 +239,6 @@ export class BackgroundSessionService {
         update: (sessionId: string, attributes: Partial<BackgroundSessionSavedObjectAttributes>) =>
           this.update(sessionId, attributes, deps),
         delete: (sessionId: string) => this.delete(sessionId, deps),
-        trackId: (searchRequest: IKibanaSearchRequest, searchId: string, options: ISearchOptions) =>
```

Comment:
I believe these are internal and shouldn't be exposed in `asScoped` since there isn't anywhere we'd be using them externally, which is why I removed them here.

Reply:
👍 
same would also make total sense on a client side